<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageQuestion</class>
 <widget class="QWidget" name="ManageQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManageQuestion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit, QPlainTextEdit, QComboBox, QRadioButton, QPushButton
{
font: 15px &quot;Times New Roman&quot;;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="frameHorizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="leftVerticalLayout">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="titleInput">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="questionLabel">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Question</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="questionInput">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="answerLabel">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Short answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="answerInput">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightVerticalLayout">
       <item>
        <widget class="QTableWidget" name="domainTableWidget">
         <property name="styleSheet">
          <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="chapterTableWidget">
         <property name="styleSheet">
          <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="questionTypeLabel">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Question type</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="questionTypeHorizontalLayout">
         <item>
          <widget class="QRadioButton" name="singleAnswerRadio">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>single answer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="multipleAnswerRadio">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>multiple answers</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="numberOfAnswersHorizontalLayout">
         <item>
          <widget class="QLabel" name="numberOfAnswersLabel">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Number of answers:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="numberOfAnswersComboBox">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="saveButtonHorizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: red;</string>
           </property>
           <property name="text">
            <string>Delete question</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Save changes</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
